controllers/article-controller.js

```diff
@@ -17,3 +17,18 @@ exports.getArticles = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (isNaN(article_id) || isNaN(inc_votes)) {

```

Comment:
This should not be a custom reject, PSQL has very specific error codes for this situation that you should be looking out for in your error handlers.